doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -881,6 +881,12 @@ Other libraries
 
   * Fixed issue where the adp536x driver was included in the immutable bootloader on Thingy:91 when :kconfig:option:`CONFIG_SECURE_BOOT` was enabled.
 
+* :ref:`mod_memfault` library:
+
+  * Added more default LTE metrics, such as band, operator, RSRP, and kilobytes sent and received.
+  * Updated the default metric names to follow the standard |NCS| variable name convention.
+
+
```

Comment:
```suggestion
```
Could remove the extra line.

Reply:
I'll do it if I need to do another push, but will leave as is if not, to avoid extra load on CI